.travis.yml

```diff
@@ -123,10 +123,20 @@ install:
         pyparsing!=2.1.6 \
         python-dateutil \
         sphinx
-    # GUI toolkits are pip-installable only for some versions of Python so don't
-    # fail if we can't install them.
-    pip install pyqt5 || true
-    python -c 'import PyQt5.QtCore' || true
+    # GUI toolkits are pip-installable only for some versions of Python so
+    # don't fail if we can't install them.  Make it easier to check whether the
+    # install was successful by trying to import the toolkit (sometimes, the
+    # install appears to be successful but shared libraries cannot be loaded at
+    # runtime, so an actual import is a better check).
+    pip install pyqt5 &&
+      python -c 'import PyQt5.QtCore' ||
+      true
```

Comment:
Can you make this something like ` && echo "PyQt5 is available" || echo "PyQt5 is not available"` (and similar below)? I know no output == okay, but it's nice to see confirmation.